pkg/pillar/types/types.go

```diff
@@ -35,10 +35,14 @@ const (
 	INSTALLED       // Available to be activated
 	BOOTING
 	RUNNING
+	PAUSING
+	PAUSED
 	HALTING // being halted
 	HALTED
 	RESTARTING // Restarting due to config change or zcli
 	PURGING    // Purging due to config change
+	BROKEN     // Domain is still alive, but its device model has failed
+	UNKNOWN    // State of the domain can't be determined
```

Comment:
You also need to add the states to the ZwSwstate function in the same file to map them to something existing to send to zedcloud. (One could add them to zedcloud, zmsg, and the UI but that takes a some coordination. So better to map to existing API states.)

Reply:
Hey @eriknordmark -- as I pointed out I don't think I need to add them just yet since these new states don't escape into being reported back to the cloud. In fact, after running additional tests it has become clear to me that we may need to solidify some of the state transition assumptions -- once that is done we can do the final mapping all the way back to the cloud.

If you're saying it is better to map them to SOMETHING now -- that's fine I can do that (since that mapping won't be used anyway).